11_circular_csimeone/scr/plot_the_driest_and_wettest_days.R

```diff
@@ -0,0 +1,142 @@
+# Download NWIS Data For UCRB
+
+library("dataRetrieval")
+library(tidyverse)
+library(lubridate)
+library(RColorBrewer)
+library(ggplot2)
+library(circular)
+library(cowplot)
+
+load("11_circular_csimeone/data_in/site_list.RData")
+
+pCd <- "00060" #Discharge, cubic feet per second, see readNWISdv help file for more options
+start_date <- as.Date("1950-04-01")
+end_date <- as.Date("2021-12-31")
+
+# for (i in 1:length(site_list)){
+#   #NWIS data pull
+#   station <- site_list[i]
+#   cat(station, i, "of", length(site_list), "\n")
+# 
+#   df <- readNWISdv(station,pCd,start_date,end_date) %>%
+#     renameNWISColumns() %>%
+#     as_tibble()
+# 
+#   write_csv(df, paste0("11_circular_csimeone/data_in/NWIS_data/", station, ".csv"))
+# }
```

Comment:
same as above, I would clean up the unused pieces that are commented out

Reply:
These were commented out so I don't download from NWIS every time. Added if statement to only do so when triggered.